src/parser/avc_parser.cpp

```diff
@@ -475,7 +485,7 @@ ParserResult AvcVideoParser::SendPicForDecode() {
         }
     }
 
-    for (i = buf_index; i < AVC_MAX_DPB_FRAMES; i++) {
+    for (i = buf_index; i < 16; i++) {
```

Comment:
why do we use `16` instead of the `AVC_MAX_DPB_FRAMES` here?

Reply:
AVC_MAX_DPB_FRAMES is increased from 16 to 18 to take care of the need to store more frames in DPB. But the ref_frames[16] in VAAPI picture parameter struct stays as the same.

Reply:
We will reduce DPB buffer size back to 16 when we implement decode/display buffer pool.